core/src/com/group66/game/screens/GameScreen.java

```diff
@@ -7,204 +7,211 @@
+	 * 
+	 * @see com.badlogic.gdx.Screen#render(float)
+	 */
 	@Override
 	public void render(float delta) {
 		/* Update the runtime */
-		runTime+=delta;
+		runTime += delta;
 
 		/* Handle input keys */
 		inputHandler.run();
 
 		Gdx.gl.glClearColor(0.2f, 0.2f, 0.3f, 1);
 		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
 
-		/*
-		shapeRenderer.begin(ShapeType.Filled);
-		shapeRenderer.setColor(Color.BLUE);
-		shapeRenderer.rect(BustaMove.BOUNCE_X_MIN, BustaMove.BOUNCE_Y_MIN,
-				BustaMove.BOUNCE_X_MAX - BustaMove.BOUNCE_X_MIN,
-				BustaMove.BOUNCE_Y_MAX - BustaMove.BOUNCE_Y_MIN);
-		shapeRenderer.end();
-		*/
-
 		game.batch.begin();
 		game.batch.enableBlending();
-		
+
 		/* Draw the background */
-		game.batch.draw(AssetLoader.bg, BustaMove.BOUNCE_X_MIN, BustaMove.BOUNCE_Y_MIN, 128, 220);
-		
+		game.batch.draw(AssetLoader.bg, Config.BOUNCE_X_MIN,
+				Config.BOUNCE_Y_MIN, Config.BOUNCE_X_MAX - Config.BOUNCE_X_MIN,
+				Config.BOUNCE_Y_MAX - Config.BOUNCE_Y_MIN);
+
 		/* Draw the balls */
 		ballManager.draw(game.batch, runTime);
-		
+
 		/* Draw the cannon */
 		cannon.draw(game.batch);
 
 		game.batch.end();
 	}
 
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.badlogic.gdx.Screen#resize(int, int)
+	 */
 	@Override
 	public void resize(int width, int height) {
 
 	}
 
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.badlogic.gdx.Screen#pause()
+	 */
 	@Override
 	public void pause() {
 
 	}
 
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.badlogic.gdx.Screen#resume()
+	 */
 	@Override
 	public void resume() {
 
 	}
 
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.badlogic.gdx.Screen#hide()
+	 */
 	@Override
 	public void hide() {
 
 	}
 
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.badlogic.gdx.Screen#dispose()
+	 */
 	@Override
 	public void dispose() {
 		// img.dispose();
 	}
 
+	/**
+	 * Setup the keys used in the game screen keys.
+	 */
+	private void setup_keys() {
+		// Setup the game keys
+		inputHandler.registerKeyMap("Shoot", Keys.SPACE);
+		inputHandler.registerKeyMap("Shoot", Keys.BACKSPACE);
+		inputHandler.registerKeyMap("Aim Left", Keys.A);
+		inputHandler.registerKeyMap("Aim Left", Keys.LEFT);
+		inputHandler.registerKeyMap("Aim Right", Keys.D);
+		inputHandler.registerKeyMap("Aim Right", Keys.RIGHT);
+		inputHandler.registerKeyMap("Place Ball", Keys.ENTER);
+
+		/* Register key names to functions */
+		inputHandler.registerKeyPressedFunc("Aim Left",
+				new InputHandler.KeyCommand() {
+					public void runCommand() {
+						cannon.cannonAimAdjust(Config.CANNON_AIM_DELTA);
+					}
+				});
+
+		inputHandler.registerKeyPressedFunc("Aim Right",
+				new InputHandler.KeyCommand() {
+					public void runCommand() {
+						cannon.cannonAimAdjust(-1f * Config.CANNON_AIM_DELTA);
+					}
+				});
+
+		inputHandler.registerKeyJustPressedFunc("Shoot",
+				new InputHandler.KeyCommand() {
+					public void runCommand() {
+						ballManager.shootRandomBall();
+					}
+				});
+	}
+
+	/**
+	 * Load a test level.
+	 */
 	private void loadLevel() {
```

Comment:
I think this should be a separate class. Also I still think its way nicer to have a json(or lookalike) file format. In that case you can also easily specify the color or noball.